skedulr/app/models/business.rb

```diff
@@ -1,3 +1,11 @@
 class Business < ApplicationRecord
   has_many :employees
+
+  def self.search(term)
+    if term
+      where('name LIKE ?', "%#{term}%")
```

Comment:
This search is a good start, one slight tweak: After you've made sure you're lowercasing in the before_save method be sure to lowercase the search term here. Later we can call this search with ajax